pkg/bootstrap/config.go

```diff
@@ -484,6 +484,7 @@ func extractAttributesMetadata(envVars []string, plat platform.Environment, meta
 		case "ISTIO_METAJSON_LABELS":
 			m := jsonStringToMap(val)
 			if len(m) > 0 {
+				meta.IstioMetaLabels = m
```

Comment:
We need to distinguish which labels are from `ISTIO_METAJSON_LABELS` and which are from pod labels. 

This is easy to handle pod label removal. FYI: https://github.com/istio/istio/pull/40036#discussion_r926894456

Reply:
I reworked this by inferring it.   `IstioMetaLabels `= meta.Labels minus  `GetProxyWorkloadLabels`

So do not need client to send duplicate labels